src/components/ActionButtonBar.vue

```diff
@@ -87,6 +87,7 @@ export default class ActionButtonBar extends Vue {
   }
   buildSurveyFile(): string {
     return JSON.stringify({
+      version: this.$store.state.version,
```

Comment:
This is so when it creates the JSON file it can store the version number referenced in the store.state object and then  stringifying and turning it from an object into a string so that file can load it properly.